docs/reference-docs/customizing_frontend/theming.md

```diff
@@ -0,0 +1,81 @@
+# Theming
+
+## Customizing the theme
+
+The Workflow Orchestrator frontend ships with a default theme leveraging the theming mechanism of Elastic UI. This theme can be partially adjusted or completely overridden.
+
+As part of the boilerplate code, the `_app.tsx` file applies a `defaultOrchestratorTheme` object to the EuiProvider.
+
+```tsx
+// _app.tsx
+
+<SeveralProviders>
+    <EuiProvider
+        colorMode={themeMode}
+        modify={defaultOrchestratorTheme}
+    >
+        ...
+    </EuiProvider>
+</SeveralProviders>
+```
+
+The default defaultOrchestratorTheme object are adjustments of the standard theme provided by Elastic UI and can be imported from the [@orchestrator-ui/orchestrator-ui-components](https://www.npmjs.com/package/@orchestrator-ui/orchestrator-ui-components) package.
```

Comment:
objects

Reply:
It is one object, but the sentence can be improved indeed